src/gemdat/rotations.py

```diff
@@ -24,15 +82,35 @@ class Orientations:
         Type of the satellite atoms
     nr_central_atoms: int
         Number of central atoms, which corresponds to the number of cluster molecules
-    nr_ligands: optional[int]
-        Number of ligands
     """
     trajectory: Trajectory
 
     center_type: str
     satellite_type: str
     nr_central_atoms: int
-    normalize_trajectory: bool = False
+
+    def __post_init__(self):
+        """Computes trajectories of unit vectors defined as the distance
+        between a central and satellite atoms, meant to track orientation of
+        molecules or clusters."""
+        lattice = self.trajectory.lattice

```

Comment:
I'm a bit confused why this is a numpy array, shouldn't this return a pymatgen `Lattice` object?

```suggestion
        lattice = self.trajectory.get_lattice()
```